api/hub/src/api.ts

```diff
@@ -236,11 +220,11 @@ api.post('/moderation/decisions/pending', async (ctx: koa.Context, next: () => P
     req.offset,
     req.limit,
   );
-  const list = await Promise.all(
-    decisions.map(decision => {
-      return getFinalDecision(decision.contentID);
-    }),
-  );
+  const list = [];
+  for (const decision of decisions) {
+    console.log('\n\nPENDING:\n\n', decision);
```

Comment:
replace console logs with logger from helpers

Reply:
I meant to remove it but forgot. It's gone now.